docs/_includes/navbar.html

```diff
@@ -49,15 +49,15 @@
 	  <ul class="nav navbar-nav">
 
 	    <li>
-	      <a href="index.html" class="{% if page.url contains 'index.html' %}active{% endif %}">Documentation</a>
+	      <a href="/index.html" class="{% if page.url contains 'index.html' %}active{% endif %}">Documentation</a>
```

Comment:
We could set the `<base href="http://ci.apache.org/projects/flink/.../">` in the header. That way, we would be able to use relative links. However, hard-coding the link to the docs in the template variables, is also not a very nice.